dcolor/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .dcolor import DColor
+from .hsvcolor import DColor as DColorHSV
+from .hsvcolor import DColor as DColorRGB
```

Comment:
should this be:
```suggestion
from .rgbcolor import DColor as DColorRGB
```

Reply:
Whoops. I fixed that in some other branches I've been working on.
...that said, I actually ended up retooling having them in separate classes entirely, then spent entirely too much time working on getting nice curves for an extra plotting variant. I'll have separate requests for them.